src/analyses/constant_propagator.cpp

```diff
@@ -623,34 +630,34 @@ void constant_propagator_ait::replace(
 
     if(it->is_goto() || it->is_assume() || it->is_assert())
     {
-      d.partial_evaluate(it->guard, ns);
+      constant_propagator_domaint::partial_evaluate(d.values, it->guard, ns);
```

Comment:
I don't think you need the `constant_propagator_domaint::` qualifier since this method lives in the same namespace

Reply:
I don't think so - it's here being invoked from `constant_propagator_ait::replace` (note that I've made `partial_evaluate` a static member of `constant_propagator_domaint`).

Reply:
Ah sorry, misread